src/Symfony/Component/Emoji/Resources/data/github-emoji.php

```diff
@@ -1813,4 +1354,506 @@
     ':zipper_mouth_face:' => '🤐',
     ':zombie:' => '🧟',
     ':zzz:' => '💤',
+    ':afghanistan:' => '🇦🇫',
```

Comment:
what is actually the sorting done on this file ? It seems we have _multiple_ rounds of the alphabet in it (each of them being sorted, indeed).

Reply:
first by decreasing emoji length, then alpha order

Reply:
for those `-emoji` files that have emojis as values, is the emoji length important ? If not, sorting only in alpha order will make it easier to compare emoji lists of different providers to see which aliases are not present in some of them, or which aliases have a different emoji associated. Both comparisons are much harder if the alias can move in the list when changing its value.

Reply:
all maps are sorted exactly the same way so that they can easily be compared
the order might matter if we switch to eg str_replace / translit one day so it's fine as is IMHO